File: Exercícios/Capitulo_2/Propostos/q8.py
""" 8. Construa um programa que receba do usuário o valor do 
salário mínimo atual. Na sequência, o programa deve solicitar 
que o usuário informe o valor do seu salário mensal. Ao fim, o 
programa deve calcular a quantidade de salários mínimos re
cebidos pelo usuário. """

sal_min = float(input("Salário mínimo atual = "))
sal_mensal = float(input("Salário mensal = "))

print(f"Você recebe {sal_mensal//sal_min} salário(s) mínimo(s) ao mês")

File: Exercícios/Capitulo_2/Propostos/q8.cpp
/*""" 8. Construa um programa que receba do usuário o valor do
salário mínimo atual. Na sequência, o programa deve solicitar
que o usuário informe o valor do seu salário mensal. Ao fim, o
programa deve calcular a quantidade de salários mínimos re
cebidos pelo usuário. """*/
#include <iostream>
using namespace std;

int main()
{
    float sal_min, sal_mensal;

    cout << "Valor do salario minimo = ";
    cin >> sal_min;

    cout << "\nValor do seu salario mensal = ";
    cin >> sal_mensal;

    cout << "Você recebe " << sal_mensal / sal_min << " salario(s) minimo(s) ao mes.";

    return 0;
}